File: app/plantilla/src/app/demo/models/EmpleadoViewModel.ts
export class Empleado {
    empl_Id?: number;
    empl_Identidad?: string;
    empl_Nombre?: string;
    empl_Apellido?: string;
    empl_Sexo?: string;
    empl_Correo?: string;
    esta_Id?: number;
    muni_Codigo?: string;
    carg_Id?: number;
    carg_Descripcion?:string;
    esta_Descripcion?:String;
    muni_Descripcion?:String;
    empl_Usua_Creacion?: number;
    empl_Fecha_Creacion?: Date;
    empl_Usua_Modifica?: number;
    empl_Fecha_Modifica?: Date;
}

export class EmpleadoDDL{
    empl_Id?: number;
    empl_Nombre?: string;
}



export class dropEmpleado{
    value?:String;
    text?:String;
}

export class EmpleadoEnviar {
    Empl_Id:string;
    empl_Identidad: string;
    Empl_Nombre: string;
    Empl_Apellido: string;
    Empl_Sexo: string;
    Empl_Correo:string;
    Muni_Codigo: string;
    Esta_Id: string;
    Carg_Id: string;
  
  


}

export class Fill {
    empl_Id: string;
    empl_Nombre: string;
    empl_Apellido?:String;
    empl_Identidad?:String;
    empl_Sexo?:String;
    muni_Codigo?:String;
    esta_Id?:String;
    carg_Id :string;
    carg_Descripcion?:string;
    esta_Descripcion?:String;
    muni_Descripcion?:String;
    dept_Descripcion?:String;
    empl_Correo:string;

    dept_Codigo?:String;
    // depa_Departamento?:String;
    empl_Usua_Creacion: string;
    empl_Usua_Modifica: string;
    empl_Fecha_Creacion : string;
    empl_Fecha_Modifica : string;
}

File: api/Proyecto_BK/Proyecto_BK.Common/Models/EmpleadoViewModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Proyecto_BK.Common.Models
{
    public class EmpleadoViewModel
    {

        public int Empl_Id { get; set; }
        public string Empl_Identidad { get; set; }
        public string Empl_Nombre { get; set; }
        public string Empl_Apellido { get; set; }
        public string Empl_Sexo { get; set; }
        public string Empl_Correo { get; set; }
        public int? Esta_Id { get; set; }
        public string Muni_Codigo { get; set; }
        public int? Carg_Id { get; set; }
        public int? Sucu_Id { get; set; }
        public int? Empl_Usua_Creacion { get; set; }
        public DateTime? Empl_Fecha_Creacion { get; set; }
        public int? Empl_Usua_Modifica { get; set; }
        public DateTime? Empl_Fecha_Modifica { get; set; }
        public bool? Empl_Estado { get; set; }

        [NotMapped]
        public string Esta_Descripcion { get; set; }
        [NotMapped]
        public string Muni_Descripcion { get; set; }
        [NotMapped]
        public string Carg_Descripcion { get; set; }
        [NotMapped]
        public string Dept_Codigo { get; set; }
        [NotMapped]
        public string Dept_Descripcion { get; set; }
        [NotMapped]
        public string Sucu_Descripcion { get; set; }
        [NotMapped]
        public string Usua_Creacion { get; set; }
        [NotMapped]
        public string Usua_Modifica { get; set; }
    }
}
